app/app.py

```diff
@@ -0,0 +1,190 @@
+import os
+import pickle
+import tempfile
+import uuid
+import dash_bootstrap_components as dbc
+import dash_uploader as du
+from dash import Dash
+from dash import Input
+from dash import Output
+from dash import clientside_callback
+from dash import dcc
+from dash import html
+
+
+dbc_css = "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
+app = Dash(__name__, external_stylesheets=[dbc.themes.UNITED, dbc_css, dbc.icons.FONT_AWESOME])
+
+# ------------------ Nav Bar ------------------ #
+color_mode_switch = html.Span(
+    [
+        dbc.Label(className="fa fa-moon", html_for="color-mode-switch"),
+        dbc.Switch(
+            id="color-mode-switch",
+            value=False,
+            className="d-inline-block ms-1",
+            persistence=True,
+        ),
+        dbc.Label(className="fa fa-sun", html_for="color-mode-switch"),
+    ],
+    className="p-2",
+)
+
+# Define the navigation bar
+navbar = dbc.Row(
+    dbc.Col(
+        dbc.NavbarSimple(
+            children=[
+                dbc.NavItem(dbc.NavLink("Doc", href="https://nplinker.github.io/nplinker/latest/")),
+                dbc.NavItem(
+                    dbc.NavLink("About", href="https://github.com/NPLinker/nplinker-webapp"),
+                ),
+                dbc.NavItem(
+                    color_mode_switch,
+                    className="mt-1 p-1",
+                ),
+            ],
+            brand="NPLinker Webapp",
+            brand_href="https://github.com/NPLinker/nplinker-webapp",
```

Comment:
```suggestion
```
Clicking it will make what users have uploaded and chose gone, so it's better not to use this `brand_href`.